.eslintrc.json

```diff
@@ -33,7 +33,8 @@
         "destructuredArrayIgnorePattern": "^_"
       }
     ],
-    "react/prop-types": "off"
+    "react/prop-types": "off",
+    "curly": ["error", "all"]
```

Comment:
I wish it had a rule for early return. https://eslint.org/docs/latest/rules/curly

Reply:
ah, yeah. There are other options, but I don't currently see a way to separate early returns

Reply:
multi is sorta similar: https://eslint.org/docs/latest/rules/curly#multi